doc/user/content/sql/create-source/webhook.md

```diff
@@ -364,6 +364,9 @@ After the source is created, you can connect with EventBridge. Follow
 using the shared secret specified in the `CHECK` statement (`x-mz-api-key`) as
 the **API key name** for request validation.
 
+## Disallowed characters
+Periods (dots, .) are not allowed in database, schema, or webhook names to prevent issues in the webhook destination url parsing.
```

Comment:
To make this as non-onerous as possible, I think we want to prohibit names that are literally `.` or `..`, but otherwise allow dots in names—e.g., `foo.bar` is a totally fine name, right?

```suggestion
Webhook sources, schemas containing webhook sources, and databases containing webhook sources may not be named `.` and `..`, as these names would not be representable in the webhook's URL.
```

Reply:
Yes, I will update to limit to names that are explicitly '.' or '..'.